test/integration-tests/ui/PackageDependencyProvider.test.ts

```diff
@@ -13,107 +13,184 @@
-        expect(dep).to.not.be.undefined;
-        expect(dep?.location).to.equal("https://github.com/swiftlang/swift-markdown.git");
-        expect(dep?.path).to.equal(
-            `${testAssetPath("dependencies")}/.build/checkouts/swift-markdown`
-        );
-    });
-
-    test("Includes local dependency", async () => {
-        const items = await treeProvider.getChildren();
-
-        const dep = items.find(n => n.name === "defaultpackage") as PackageNode;
-        expect(dep).to.not.be.undefined;
-        expect(dep?.location).to.equal(testAssetPath("defaultPackage"));
-        expect(dep?.path).to.equal(testAssetPath("defaultPackage"));
-    });
-
-    test("Lists local dependency file structure", async () => {
-        const items = await treeProvider.getChildren();
-
-        const dep = items.find(n => n.name === "defaultpackage") as PackageNode;
-        expect(dep).to.not.be.undefined;
-
-        const folders = await treeProvider.getChildren(dep);
-        const folder = folders.find(n => n.name === "Sources");
-        expect(folder).to.not.be.undefined;
-
-        expect(folder?.path).to.equal(`${testAssetPath("defaultPackage")}/Sources`);
-
-        const childFolders = await treeProvider.getChildren(folder);
-        const childFolder = childFolders.find(n => n.name === "PackageExe");
-        expect(childFolder).to.not.be.undefined;
-
-        expect(childFolder?.path).to.equal(`${testAssetPath("defaultPackage")}/Sources/PackageExe`);
-
-        const files = await treeProvider.getChildren(childFolder);
-        const file = files.find(n => n.name === "main.swift");
-        expect(file).to.not.be.undefined;
-
-        expect(file?.path).to.equal(
-            `${testAssetPath("defaultPackage")}/Sources/PackageExe/main.swift`
-        );
-    });
-
-    test("Lists remote dependency file structure", async () => {
-        const items = await treeProvider.getChildren();
-
-        const dep = items.find(n => n.name === "swift-markdown") as PackageNode;
-        expect(dep).to.not.be.undefined;
-
-        const folders = await treeProvider.getChildren(dep);
-        const folder = folders.find(n => n.name === "Sources");
-        expect(folder).to.not.be.undefined;
-
-        const path = `${testAssetPath("dependencies")}/.build/checkouts/swift-markdown`;
-        expect(folder?.path).to.equal(`${path}/Sources`);
-
-        const childFolders = await treeProvider.getChildren(folder);
-        const childFolder = childFolders.find(n => n.name === "CAtomic");
-        expect(childFolder).to.not.be.undefined;
-
-        expect(childFolder?.path).to.equal(`${path}/Sources/CAtomic`);
-
-        const files = await treeProvider.getChildren(childFolder);
-        const file = files.find(n => n.name === "CAtomic.c");
-        expect(file).to.not.be.undefined;
-
-        expect(file?.path).to.equal(`${path}/Sources/CAtomic/CAtomic.c`);
+    suite("Full work flow tests", function () {
```

Comment:
"full workflow", i.e. multiple paths and actions, is more smoke level. Find to group a feature in a suite, but let's make sure each test case is one path.

Reply:
Done.